scala/lambdas/preingest-tdr-package-builder/src/test/scala/uk/gov/nationalarchives/preingesttdrpackagebuilder/LambdaTest.scala

```diff
@@ -85,7 +87,26 @@ class LambdaTest extends AnyFlatSpec with ScalaCheckDrivenPropertyChecks:
     filePath <- Gen.nonEmptyListOf(Gen.nonEmptyStringOf(Gen.alphaChar)).suchThat(_.size >= 2).map(s => s"/${s.mkString("/")}")
     (potentialTdrRef, potentialDriBatchRef) <- tdrOrDriBatchGen
     description <- Gen.option(Gen.nonEmptyStringOf(Gen.asciiChar))
-  } yield TestData(fileId, series, body, date, potentialTdrRef, fileName, fileSize, List(checksum), fileRef, groupId, batchId, filePath, potentialDriBatchRef, description)
+    sortOrder <- Gen.choose(0, 10)
```

Comment:
Could you not do `Gen.option(Gen.choose(0, 10))` instead of `Option(sortOrder)` on line 107?